### PR TITLE
Updated Geant4 physics lists

### PR DESCRIPTION
**Pull request by civanch**
Updated alternative Geant4 Physics Lists which may be used for various studies. 8 obsolete lists removed, 3 new added. Now most part of alternative Physics Lists use only components from geant4 libraries instead of CMS custom. 

Local EM physics builder are updated according to recent modifications in Geant4. 

Header files of CMS Physics Lists are renamed according to the CMS code conversion (.h instead of .hh).

Modifications better correspond to the coming new Geant4 10.4 but are working with the current Geant4 10.2.

The default Physics List FTFP_BERT_EMM is not changed, so unchanged results are expected.

### CODE REVIEW

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

File: SimG4Core/PhysicsLists/plugins/QGSPCMS_FTFP_BERT_EMY.hh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#ifndef SimG4Core_PhysicsLists_QGSPCMS_FTFP_BERT_EMY_H
```
Comment (davidlange6) [inline SimG4Core/PhysicsLists/plugins/QGSPCMS_FTFP_BERT_EMY.hh]: hi @civanch - cms convention is .h not .hh

Comment (civanch) [reply SimG4Core/PhysicsLists/plugins/QGSPCMS_FTFP_BERT_EMY.hh]: Hi @davidlange6 , it is historical in this sub-library. Can be changed....

**Comment by davidlange6** [comment]
i think that would be great..

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**